next-sitemap.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  siteUrl: process.env.SITE_URL || 'https://mycovidstory.ca',
```

Comment:
I also need the deploy URL for my metadata PR - I think we can do `process.env.NEXT_PUBLIC_VERCEL_URL` here, as long as we set an env var (in both our local `.env` and in Vercel's env var settings) that looks like:

`NEXT_PUBLIC_VERCEL_URL=VERCEL_URL`

*[ Docs on NextJS env vars](https://nextjs.org/docs/basic-features/environment-variables#exposing-environment-variables-to-the-browser) (esp how they get exposed to the browser with the `NEXT_PUBLIC_` prefix
* [Docs on Vercel default env vars](https://vercel.com/docs/environment-variables#system-environment-variables) (specifically I think `VERCEL_ENV` and `VERCEL_URL` are going to be useful for us.